src/App.js

```diff
@@ -0,0 +1,18 @@
+import './styles/App.css';
+import { Routes, Route } from 'react-router-dom';
+import Layout from './components/Layout';
+import BooksPage from './routers/bookPage';
+import AuthorPage from './routers/authorPage';
+
+function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route path="/" element={<BooksPage />} />
+        <Route path="/authors" element={<AuthorPage />} />
```

Comment:
- [ ] This should navigate to Categories instead of Author. You can implement the Author section after you completed all these project milestones but please for now stick to the requirement of this project. 

```javascript
<Route path="/categories" element={<Categories />} />
```